src/components/FilterPanel.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+
+const FilterPanel = ({
+	data,
+	applyFilters,
+	updateFilter,
+	filters,
+	maxLimit
+}) => {
+	const [filterOptions, setFilterOptions] = useState([]);
+	useEffect(() => {
+		if (data.filters) setFilterOptions(data.filters.split(','));
+	}, []);
+	return (
+		<div className="filter-panel">
+			<div className="title">Filter Panel</div>
+			<div className="filter-container">
+				<p>Test Correction:</p>
+				<div className="control">
+					<select
+						name="correction"
+						value={filters['correction']}
+						onChange={updateFilter}
+					>
+						{['Holm-Bonferroni', 'Benjamini Hochberg', 'Bonferroni'].map(
```

Comment:
I think it would be a good idea to have this array defined at the top level (above `const FilterPanel`) to make it obvious it's hardcoded and can be changed. Same with `[0.05, 0.5, 1.0]` below.